test/io/configs/no-defaults.config

```diff
@@ -1,3 +1,4 @@
+cors-allowed-origins = "http://example.com"
```

Comment:
@taimoorzaeem It would be very useful to have this as an [in-database config](https://postgrest.org/en/stable/references/configuration.html#in-database-configuration) too.

Reply:
Also, I think I might have given the config a bad name on the issue. My bad. We have the `server-` prefix for all server related configs. So it would make more sense as `server-cors-allowed-origins`, then in-db `server_cors_allowed_origins`.